src/modules/launcher/Plugins/Microsoft.Plugin.Uri/Main.cs

```diff
@@ -4,42 +4,31 @@
 
 using System;
 using System.Collections.Generic;
-using System.IO.Abstractions;
-using System.Text;
 using ManagedCommon;
 using Microsoft.Plugin.Uri.UriHelper;
 using Wox.Infrastructure;
 using Wox.Infrastructure.Storage;
 using Wox.Plugin;
-using Wox.Plugin.Logger;
 
 namespace Microsoft.Plugin.Uri
 {
-    public class Main : IPlugin, IPluginI18n, IContextMenu, ISavable, IDisposable
+    public class Main : IPlugin, IPluginI18n, IContextMenu, ISavable, IReloadable, IDisposable
     {
-        private static readonly IFileSystem FileSystem = new FileSystem();
-        private static readonly IPath Path = FileSystem.Path;
-        private static readonly IFile File = FileSystem.File;
-
         private readonly ExtendedUriParser _uriParser;
         private readonly UriResolver _uriResolver;
         private readonly PluginJsonStorage<UriSettings> _storage;
         private bool _disposed;
         private UriSettings _uriSettings;
-        private RegistryWrapper _registryWrapper;
 
         public Main()
         {
             _storage = new PluginJsonStorage<UriSettings>();
             _uriSettings = _storage.Load();
             _uriParser = new ExtendedUriParser();
             _uriResolver = new UriResolver();
-            _registryWrapper = new RegistryWrapper();
         }
 
-        public string BrowserIconPath { get; set; }
-
-        public string BrowserPath { get; set; }
+        internal BrowserInfo Browser { get; private set; }
```

Comment:
Why do you changed this only in this plugin to internal. Can't we keep both plugins behave the same?